stdlib/LinearAlgebra/src/hessenberg.jl

```diff
@@ -90,6 +90,10 @@ Base.isassigned(H::UpperHessenberg, i::Int, j::Int) =
 Base.@propagate_inbounds getindex(H::UpperHessenberg{T}, i::Integer, j::Integer) where {T} =
     i <= j+1 ? convert(T, H.data[i,j]) : zero(T)
 
+Base._reverse(A::UpperHessenberg, dims) = reverse(Matrix(A); dims)
+Base._reverse(A::UpperHessenberg, dims::Colon) = (reverse(Matrix(A)), A.data)
+Base._reverse!(A::UpperHessenberg, dims::Colon) = (reverse!(A.data); A)
```

Comment:
```suggestion
```
This method is incorrect and should be deleted — `reverse(A)` is "lower-Hessenberg", which we don't have a type for and even if we did have a type it can't be generated in-place.